model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/Constants.java

```diff
@@ -18,5 +18,6 @@
 
 public interface Constants {
     public static final Integer SUPPORTED_VERSION_CLIENT = 1;
+    public static final Integer SUPPORTED_VERSION_CLIENT_SCOPE = 1;
     public static final Integer SUPPORTED_VERSION_ROLE = 1;
```

Comment:
Is this constant name really intuitive? Do we support only one version? Wouldn't CURRENT_VERSION_X be better?

Reply:
It's probably not very intuitive. There is only one version supported for each area in each keycloak. What about `CURRENT_SCHEMA_VERSION_X`?

Reply:
`CURRENT_SCHEMA_VERSION_X` looks good to me.

Reply:
resolved, thanks!